subsys/driver_handlers/eth.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <syscall_handler.h>
+#include <eth.h>
+
+_SYSCALL_HANDLER(eth_send, dev, buffer, len_p)
```

Comment:
Is this in anticipation for running the network layer as a user thread? I don't see much use otherwise.

Reply:
My hope is we can get to that point someday yes. But as above, I can just drop this subsystem from the PR.